cmd/bbolt/main.go

```diff
@@ -1264,13 +1293,13 @@ func (cmd *GetCommand) Run(args ...string) error {
 		}
 
 		// Find value for given key.
-		val := lastbucket.Get([]byte(key))
+		val := lastbucket.Get(key)
 		if val == nil {
-			return ErrKeyNotFound
+			return fmt.Errorf("Error %w for key: %q hex: \"%x\"", ErrKeyNotFound, key, string(key))
```

Comment:
Suggest to output the key in exactly the same format as inputted by user.

Reply:
I wanted to provide very explicit feedback. In particular usually we  (etcd) want to pass \x00\x00... starting strings... and char-0 does not passes well through linux's `vexec` command so the strings are truncated. 

Both format's allow to easily cross check against `bbolt page db 5`

```
$ echo $'aaa\x00aaa' | hexdump 

0000000 6161 0a61                              
0000004
```